level-2/행렬테두리회전하기/0imbean0.js

```diff
@@ -0,0 +1,61 @@
+function getSquare(rows, columns) {
+  return Array.from({ length: rows }).map((_, row) => {
```

Comment:
`Array.from()` 의 경우 다음과 같이 정의되어 있어요!

```js
Array.from(arrayLike[, mapFn[, thisArg]])
```

map을 별도로 선언하지 않고 다음과 같이 사용하셔도 좋을거 같아요!

```suggestion
  return Array.from({ length: rows }, ((_, row) => {})
```